amr-wind/overset/overset_ops_K.H

```diff
@@ -0,0 +1,119 @@
+#ifndef OVERSET_OPS_K_H_
+#define OVERSET_OPS_K_H_
+
+#include <AMReX_FArrayBox.H>
+
+namespace amr_wind::overset_ops {
+
+// Approximate signed distance function
+amrex::Real AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+asdf(const amrex::Real a_vof, const amrex::Real i_th, const amrex::Real tiny)
+{
+    // function of local vof value and interface thickness
+    return (i_th * log((a_vof + tiny) / (1. - a_vof + tiny)));
+}
+
+amrex::Real AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE alpha_flux(
+    int i,
+    int j,
+    int k,
+    int dir,
+    const amrex::Real margin,
+    amrex::Array4<amrex::Real const> const& vof,
+    amrex::Array4<amrex::Real const> const& tg_vof,
+    amrex::Array4<amrex::Real const> const& normal)
+{
+    // Set up neighbor indices
+    int ii = i;
```

Comment:
make i/j/k into intvect iv and then do ops on that `ivm` etc and then the rest of the code will look much cleaner

Reply:
didn't know I could call fields with intvect! now it seems obvious.